microsoft/vasim
# Tuning

# Pull Request

Add tuning functionality to web ui (Streamlit)
---

## Description

You can  input config modifications to run tuning 

---

## Type of Change

- ✨ New feature  

---

## Testing
1. Run
```
streamlit run examples/streamlit/web_demo.py
```
2. Check you UI with localhost
3. Run tuning

---

## Additional Notes (optional)

There is a bug https://github.com/microsoft/vasim/issues/119 

The issues is that when you run tuning there is a result table, where it should be config params and metrics. However now there are no configs, just metrics, which is not expected behavior.
